src/CMakeLists.txt

```diff
@@ -424,6 +424,15 @@ if(WITH_BLUESTORE_PMEM OR WITH_RBD_RWL)
   endif()
 endif()
 
+if(WITH_IO_URING)
+  if(WITH_SYSTEM_URING)
+    find_package(uring REQUIRED)
+  else()
+    include(Builduring)
+    build_uring()
+  endif()
+endif()
```

Comment:
i'd suggest move this chunk of code down to `src/os/CMakeLists.txt`.

Reply:
OK